packages/clerk-js/src/ui/utils/phoneUtils.ts

```diff
@@ -115,3 +116,34 @@ export function stringToFormattedPhoneString(str: string): string {
   const parsed = parsePhoneString(str);
   return `+${parsed.code} ${formatPhoneNumber(parsed.number, parsed.pattern, parsed.code)}`;
 }
+
+export const sortByPriority = (a: CountryEntry, b: CountryEntry) => {
+  if (a.priority > b.priority) {
+    return -1;
+  }
+  if (a.priority < b.priority) {
+    return 1;
+  }
+  return 0;
+};
+
+export function getLongestValidCountryCode(value: string) {
+  let result = '';
+  let countries = null;
```

Comment:
Lets initialise this inside the for block directly, since its not used by anything else right now